packages/core/src/rules/line_break_multiple_parameters.ts

```diff
@@ -55,6 +57,12 @@ export class LineBreakMultipleParameters extends ABAPRule {
         let previous = parameters[0];
         for (let i = 1; i < parameters.length; i++) {
           const current = parameters[i];
+
+          if (i < this.conf.count) {
```

Comment:
the unit tests use the default configuration, I'm not sure what happens if `conf.count` is undefined, suggest the follwing

```suggestion
          if (this.conf.count && i < this.conf.count) {
```